front-end-admin/src/components/pages/Home.vue

```diff
@@ -1,6 +1,11 @@
 <template>
   <div>
-    <login />
+    <div v-if="!loggedIn">
```

Comment:
Instead of placing this in Home.vue, you'll want to place this in the App.vue file located in the components folder. That's where the navbar and login/logout are at in order to be on every page.